models/index.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+var fs        = require('fs');
```

Comment:
Why do you use obsolete `var`'s here?

Reply:
It happens because of using sequelize-cli. This files (models, seeds, migrations) are automatic-generated.